core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.spec.ts

```diff
@@ -83,10 +83,16 @@ describe('Side Navigation Bar Component', () => {
     expect(componentInstance.currentUrl).toEqual(currentUrl);
   });
 
-  it('should toggle classroom submenu', () => {
-    componentInstance.classroomSubmenuIsShown = false;
-    componentInstance.toggleClassroomSubmenu();
-    expect(componentInstance.classroomSubmenuIsShown).toBeTrue();
+  it('should toggle learn submenu', () => {
+    componentInstance.learnSubmenuIsShown = false;
+    componentInstance.togglelearnSubmenu();
+    expect(componentInstance.learnSubmenuIsShown).toBeTrue();
```

Comment:
Just for completeness add a check if you toggle it again, that it should be false (here and below)

Reply:
done!